sdk/keyvault/ci.yml

```diff
@@ -38,9 +38,13 @@ stages:
     ServiceDirectory: keyvault
     ArtifactName: packages
     Artifacts:
-    - name: Azure.Security.KeyVault.Certificates
-      safeName: AzureSecurityKeyVaultCertificates
-    - name: Azure.Security.KeyVault.Keys
-      safeName: AzureSecurityKeyVaultKeys
-    - name: Azure.Security.KeyVault.Secrets
-      safeName: AzureSecurityKeyVaultSecrets
+    - name: Microsoft.Azure.KeyVault
```

Comment:
Why are you removing the track 2 libraries? Did you intend to add the track 1 libraries or did you want to create another parallel pipeline for the track 1? I would suggest the later which is what we did for keyvault in java (https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/ci.data.yml). 

FYI @mitchdenny

Reply:
Yes, this change was intentional. We want to only publish track 1 packages with this new message to hopefully direct developers to track 2 packages.

Reply:
We generally don't want to edit this ci.yml file each time we release a set of track 1 or track 2 libraries we should either list them both and only approve the ones you want to release or create a separate pipeline for track 1. 

With that said if this is the last time we expect to release the track 1 packages and you plan to revert this back then I'm fine with this. Hopefully when you revert this back you can also remove the track 1 code from master.

Reply:
Yes, I will revert. We don't want the release pipeline to prompt whether to publish these or not each time. Too easy for someone to mistakenly publish them. If, however, DevOps gave guidance - such as conditioning out track 1 packages by default with a build-time variable to opt into publishing them - I could make such changes.

Reply:
So if you or @AlexGhiondea could sign off, I'll resolve this without merging and push to a feature branch in the upstream repo, kick off a release build, revert the ci.yml change, then submit a PR to master.